pkg/kubelet/util/sliceutils/sliceutils.go

```diff
@@ -53,6 +53,9 @@ func (s PodsByCreationTime) Less(i, j int) bool {
 type ByImageSize []kubecontainer.Image
```

Comment:
I think I'm fine with us keeping `ByImageSize` as the name... in my mind, that's the "public" behavior we care about... the ID tie-breaker feels like an implementation detail.